src/AaveHelper.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8;
+
+import {IAavePool} from "./vendored/IAavePool.sol";
+import {IERC20} from "./vendored/IERC20.sol";
+import {SafeERC20} from "./vendored/SafeERC20.sol";
+
+/// @title AaveHelper
+/// @author CoW DAO developers
+/// @notice A helpful helper.
+contract AaveHelper {
+    using SafeERC20 for IERC20;
+
+    address public constant AAVE_LENDING_POOL = 0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2;
```

Comment:
nit: this should probably be part of the hook data (since Aave can have multiple instances on the same chain)